apps/src/templates/levelSummary/FreeResponseResponses.jsx

```diff
@@ -2,13 +2,19 @@ import PropTypes from 'prop-types';
 import React from 'react';
 
 import Alert from '@cdo/apps/componentLibrary/alert/Alert';
+import DCDO from '@cdo/apps/dcdo';
 import i18n from '@cdo/locale';
 
 import ResponseMenuDropdown from './ResponseMenuDropdown';
 
 import styles from './summary.module.scss';
 
-const FreeResponseResponses = ({responses}) => {
+const FreeResponseResponses = ({responses, showStudentNames}) => {
+  const constructStudentName = response =>
```

Comment:
Not necessarily for this PR, but we should really have a utils function that gets called each time we need the combined name instead of doing it manually. If we already have one, let's use it here, if not it's probably some tech debt we should fix at some point.

Reply:
I will make a ticket for this!